runtest use discover. cleanup

dsentinel opened the pull request:
Use `unittest`'s discover to run all tests.
We weren't excluding or adding any anyway.

derick-hess commented:
The test that is failing is for ph5tostationxml. It is very old and needs to be rewritten. It was an original test that Nick wrote a few years ago before we had the repository fully set up. ph5tostationxml has been refactored since then. 

I would suggest just excluding that file for now and have it rewritten.

dsentinel commented:
rebased and force pushed

> The test that is failing is for ph5tostationxml. It is very old and needs to be rewritten. It was an original test that Nick wrote a few years ago before we had the repository fully set up. ph5tostationxml has been refactored since then.
> 
> I would suggest just excluding that file for now and have it rewritten.

Removed test_ph5tostationxml.py
The tests in this file were already in and belong in test_ph5utils.py
Perhaps this was copied as a template to be edited and create a new test module.
